lib/modules/apostrophe-seo-doc-type-manager/index.js

```diff
@@ -17,8 +17,21 @@ module.exports = {
           min: 50,
           max: 300,
           htmlHelp: 'A short and accurate summary of the content of the page used in search results. It should be <a href="https://moz.com/learn/seo/meta-description" target="_blank">between 50-160 characters</a>.'
-        }
-
+        },
+        {
+          name: '_seoCanonical',
+          label: 'Canonical Link',
+          type: 'joinByOne',
+          withType: 'apostrophe-page',
```

Comment:
Appears to be hardcoded to pick a page only. Contrary to documentation.

Also I am not sure this field makes sense as a join. Why would you make two copies of the content? It is much more common that the field is needed because something about the project causes the page or piece to appear in two places for programmatic reasons. The most frequent reason for that is failure to canonicalize the hostname, which we address via `--canonical` in mechanic.

If we are going to offer an Apostrophe feature for _seoCanonical, it should probably be a convenience for doing it automatically based on the `_url` of `data.piece || data.page`. If `baseUrl` is set that could be pretty useful.

I could be missing something.

Reply:
> It is much more common that the field is needed because something about the project causes the page or piece to appear in two places for programmatic reasons. 

This isn't necessarily true. You could have multiple versions for marketing purposes or the reason Brad pointed out. Google outlines a few reasons: https://support.google.com/webmasters/answer/139066?hl=en

Reply:
I do think the limit to apostrophe-pages could be an issue, though.